### PR TITLE
Update system tests schema config

### PR DESCRIPTION
**Pull request by mabuaisha**

### CODE REVIEW

File: cosmo_tester/config_schemas/cfy_cluster_manager.yaml [hunk 1]
```diff
@@ -4,10 +4,16 @@ rpm_path:
...
   description: The manager-install RPM path to install.
-  default: 'https://repository.cloudifysource.org/cloudify/5.2.0/.dev1-release/cloudify-manager-install-5.2.0-.dev1.el7.x86_64.rpm'
+  default: 'https://repository.cloudifysource.org/cloudify/5.3.0/.dev1-release/cloudify-manager-install-5.3.0-.dev1.el7.x86_64.rpm'
 5_1_0_manager_install_rpm_path:
   description: manager-install v5.1.0 RPM path to install.
   default: 'https://repository.cloudifysource.org/cloudify/5.1.0/ga-release/cloudify-manager-install-5.1.0ga.rpm'
 5_1_1_manager_install_rpm_path:
   description: manager-install v5.1.1 RPM path to install.
   default: 'https://repository.cloudifysource.org/cloudify/5.1.1/ga-release/cloudify-manager-install-5.1.1-ga.el7.x86_64.rpm'
+5_1_2_manager_install_rpm_path:
```
Comment (YoniItzhak) [inline cosmo_tester/config_schemas/cfy_cluster_manager.yaml:14]: This is not needed for now. I'll add it if needed.

File: cosmo_tester/config_schemas/cfy_cluster_manager.yaml [hunk 2]
```diff
@@ -4,10 +4,16 @@ rpm_path:
...
+  description: manager-install v5.1.2 RPM path to install.
+  default: 'https://repository.cloudifysource.org/cloudify/5.1.2/ga-release/cloudify-manager-install-5.1.2-ga.el7.x86_64.rpm'
+5_2_0_manager_install_rpm_path:
```
Comment (YoniItzhak) [inline cosmo_tester/config_schemas/cfy_cluster_manager.yaml:17]: This is not needed for now. I'll add it if needed.

File: cosmo_tester/config_schemas/cfy_manager_upgrade.yaml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 namespace: upgrade
 upgrade_version:
   description: The version to upgrade to.
-  default: '5.1.2'
+  default: '5.3.0'
```
Comment (YoniItzhak) [inline cosmo_tester/config_schemas/cfy_manager_upgrade.yaml:4]: This should stay `5.1.2`, as it tests the command `cfy_manager upgrade` and this command cannot be used with `5.3.0`.

**Review by YoniItzhak — CHANGES_REQUESTED** [review]

**Review by mateumann — CHANGES_REQUESTED** [review]
isn't it lacking (some) stuff for 5.2.0 (e.g. 5_2_0_manager_install_rpm_path)?
… also manager_image_names_centos.yaml has invalid image for "5_2_0" (I guess should be cfy-manager-premium-5.2.0)
(same comment for manager_image_names_rhel.yaml)

**Comment by mabuaisha** [comment]
@mateumann thanks for your review, I fixed the invalid image name for redhat and centos and regarding the cluster upgrade path @YoniItzhak will update that as he mentioned that in his review

**Review by mateumann — DISMISSED** [review]
stuff related to `5_2_0_manager_install_rpm_path` will be added later

File: cosmo_tester/config_schemas/manager_image_names_centos.yaml [hunk 4]
```diff
@@ -1,10 +1,22 @@
...
+  description: Name of manager image for 5.2.0. All non master images are only used for upgrade tests.
+  default: cloudify-manager-premium-5.2.0
+"5_2_0_installer":
+  description: Name of image with 5.2.0 manager installer for bootstrap or cluster installs.
   default: cfy-installer-premium-5.2.0
+"5_1_2":
+  description: Name of manager image for 5.1.2. All non master images are only used for upgrade tests.
+  default: cloudify-manager-premium-5.1.2
+"5_1_2_installer":
+  description: Name of manager image for 5.1.2. All non master images are only used for upgrade tests.
```
Comment (YoniItzhak) [inline cosmo_tester/config_schemas/manager_image_names_centos.yaml:18]: Should be changed to " Name of image with 5.1.2 manager installer for bootstrap or cluster installs."

**Review by YoniItzhak — APPROVED** [review]

**Pull request merged**